test/testdata/test-nightly-group-prom-metric-tests.yaml

```diff
@@ -0,0 +1,90 @@
+- name: up
```

Comment:
Another place a leading comment could be helpful.

Reply:
done.